whitepapers/0011_accountant.md

```diff
@@ -81,110 +128,159 @@ pub enum WormholeQuery {
+- After confidence is achieved that both accountants are running correctly in log-only mode, switch to enforcing mode per protocol as needed. As long as at least a quorum of guardians submit their observations to the accountants, only a super-minority (1/3) is needed to turn on enforcing-mode for it to be effective.
 
 ## Security Considerations
 
 ### Threat Model
 
-Since the Global Accountant is implemented as an Integrity Checker, it inherits the threat model of Integrity Checkers. Therefore, it is only able to block messages and not create or modify messages.
+Since both Global Accountants are implemented as Integrity Checkers, they inherit the threat model of Integrity Checkers. Therefore, they are only able to block messages and not create or modify messages.
+
+### Protocol Isolation
+
+The dual accountant architecture provides protocol isolation benefits:
+
+- The Base and NTT accountants can be enabled independently
+- Token Bridge exploits cannot affect NTT accounting and vice versa
+- Each protocol can have different enforcement policies
+- Issues in one accountant don't necessarily impact the other
 
 ### Susceptibility to wormchain downtime
 
-Once all token transfers are gated on approval from the accountant, the uptime of the guardian network for those transfers becomes dependent on the uptime of wormchain itself.  Any downtime for wormchain would halt all token transfers until the network is back online (or 2/3+ of the guardians choose to disable the accountant).
+Once transfers are gated on approval from the respective accountants, the uptime of the guardian network for those transfers becomes dependent on the uptime of wormchain itself. Any downtime for wormchain would halt transfers until the network is back online (or 2/3+ of the guardians choose to disable the relevant accountant).
+
+In practice, cosmos chains are stable and wormchain is not particularly complex. However, once the accountants are live, attacking wormchain could shut down transfers for both protocols.
+
+### Token Bridge and NTT messages cannot reach quorum if 7 or more, but less than 13 Guardians decide to disable Accountant
+
+Due to the nature of the accountants, a guardian that chooses to enable them in enforcing-mode will refuse to sign a transfer until it sees that the transfer is committed by the appropriate accountant. If a super-minority of guardians choose to not enable an accountant at all (i.e. stop sending their observations to the contract) then we will end up with a stalemate: the guardians with the feature disabled will be unable to reach quorum because they don't have enough signatures while the guardians in enforcing-mode will never observe the transfer being committed by the accountant (and so will never sign the observation). This is very different from [the governor feature](0007_governor.md), where only a super-minority of guardians have to enable it to be effective.
```

Comment:
Is it worth noting here that the stalemate could be avoided if the guardians enable it in log only mode rather than not enable it at all?

Reply:
Do you mean that in log only mode that the Guardians will send observations in a way that will avoid the stalemate? I wasn't aware of that if soa

Reply:
If a guardian is in log only mode, it still sends observations to the contract. It just doesn't delay approving the VAA until the contract publishes the approval. The contract will count observations from guardians running in log only mode.